.github/workflows/build-test.yml

```
@@ -148,6 +148,21 @@ jobs:
       cache-path: ~/.ivy2/local/com.typesafe.play
       cache-key: play-published-local-jdk{0}-${{ github.sha }}-${{ github.event_name != 'schedule' || github.run_id }}
 
+  scala3-compilation:
+    name: Build with Scala3
+    uses: playframework/.github/.github/workflows/cmd.yml@v2
+    with:
+      java: 11
+      scala: 3.1.2
+      # doing nothing for now, but as we progress we should add here the modules that are ready for compilation
+      # with Scala 3
+      # sbt "-Dscala.version=$SCALA_VERSION
+      #  Play / compile
+      #  Play / Test / compile
+      #  "
+      cmd: >-
+        sbt "-Dscala.version=$SCALA_VERSION"
```

For `cmd.yml@v2` should be
```suggestion
        sbt "-Dscala.version=$MATRIX_SCALA"
```

---

Thanks, force pushed again.